initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -68,7 +68,7 @@
                 enableKeyDiff="true"
                 enableSysLog="true"
                 fastLaneQueue="true"
-                fieldTranslationEnabled="payload"
+                fieldTranslationEnabled="payloadRemoveOnly"
```

Comment:
This should be consisten with testing/regress/environment.xml.in.

Reply:
The changes between initfiles/etc/DIR_NAME/environment.xml.in and testing/regress/environment.xml.in look consistent to me. I'm probably not misunderstanding what you mean.

Reply:
agreed, I missed the testing/regress change.